gen/dcos-config.yaml

```diff
@@ -875,7 +875,7 @@ package:
           ],
           "LocalCommands": [
               {
-                  "Command": ["dmesg"]
+                  "Command": ["dmesg", "-T"]
```

Comment:
@janisz mentioned that this will crash on centos6. @somikbehera @masonse Do we support this version? Do you see a problem with unconditionally adding this even if we don't "officially" support centos6 even if it will crash here?

Reply:
@klueska We do not support CentOS 6, I don't see a problem with this small change.

Reply:
👍